bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -178,44 +173,55 @@ private boolean isReachable(String ipAddress) {
         @Override
         protected void doConnectedRun() throws IOException, ApiException {
             Map<String, FullSensor> lastSensorStateCopy = new HashMap<>(lastSensorStates);
+            lastSensorStates.clear();
```

Comment:
If you clear all the states, you will loose the previous state in case it is not changed by the thing handler.

Reply:
Changed this!